### PR TITLE
Add pathlib code to directories.py sample

### PR DESCRIPTION
**Pull request by JustAShoeMaker**
Add pathlib code to directories.py sample script equivalent to `os.path.dirname(os.path.abspath(__file__)) `  and provide explicit comparison/equality to it.

### CODE REVIEW

File: doc/runtime-information.rst [hunk 1]
```diff
@@ -119,6 +120,8 @@ symbolic link::
 	print( 'sys.argv[0] is', sys.argv[0] )
 	print( 'sys.executable is', sys.executable )
 	print( 'os.getcwd is', os.getcwd() )
+	print( 'os.path.dirname(os.path.abspath(__file__)) is ', os.path.dirname(os.path.abspath(__file__)))
+	print( 'PurePath(Path(__file__).absolute()).parent is ', PurePath(Path(__file__).absolute()).parent)
```
Comment (Legorooj) [inline doc/runtime-information.rst:124]: Why `PurePath` and not `Path`? The code should work as `Path(__file__).absolute().parent`.

**Review by Legorooj — CHANGES_REQUESTED** [review]

File: doc/runtime-information.rst (hunk 1 above)
Comment (JustAShoeMaker) [inline doc/runtime-information.rst:124]:
```suggestion
	print( 'Path(Path(__file__).absolute()).parent is ', Path(Path(__file__).absolute()).parent)
```

Comment (JustAShoeMaker) [reply doc/runtime-information.rst:124]: Dont really recall now.

**Comment by Legorooj** [comment]
After considering, I've decided to reject this change. I've put this on the todo list of things when this page of the docs gets an overhaul.

**Pull request closed without merging**